backend/rp2email.py

```diff
@@ -313,55 +329,81 @@ def html_to_pdf(self, html_content: str, output_path: str) -> Path:
         import json
```

Comment:
@namtran6701 Consider moving imports to the top of the file to follow Python’s recommended practice, rather than importing inside an if block